temporal-sdk/src/main/java/io/temporal/activity/LocalActivityOptions.java

```diff
@@ -75,8 +83,8 @@ public Builder setScheduleToCloseTimeout(Duration timeout) {
     }
 
     /**
-     * Maximum time to retry locally keeping workflow task open through heartbeat. Default is 6
-     * workflow task timeout.
+     * Maximum time to retry locally, while keeping the Workflow Task open via a Heartbeat.
+     * Default value is 6 workflow task timeout.
```

Comment:
This comment is not clear.
6 what? 6 seconds? 6 minutes? 6 attempts?
Also, capitalize `Workflow Task`

Reply:
Took me ages to find this, but it's basically the `Workflow Task`'s timeout multiplied by 6.
https://github.com/temporalio/sdk-java/blob/master/temporal-sdk/src/main/java/io/temporal/internal/sync/SyncWorkflowContext.java#L373